pkg/csi/csi_ops.go

```diff
@@ -86,7 +91,11 @@ func (o *validateOperations) FetchPV(ctx context.Context, pvName string) (*v1.Pe
 	if o.kubeCli == nil {
 		return nil, fmt.Errorf("kubeCli not initialized")
 	}
-	return o.kubeCli.CoreV1().PersistentVolumes().Get(ctx, pvName, metav1.GetOptions{})
+	pv, err := o.kubeCli.CoreV1().PersistentVolumes().Get(ctx, pvName, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return pv, nil
```

Comment:
Fake clients (in k8s 1.31 client-go sdk) no longer returns nil object in case of error. So we had to add explicit checks